src/main/java/uk/gov/pay/directdebit/payments/api/PaymentRequestValidator.java

```diff
@@ -15,19 +17,24 @@
     private final static int MIN_AMOUNT = 1;
     private final static int MAX_AMOUNT = 10000000;
 
-    private final static Map<String, Function<String, Boolean>> validators = ImmutableMap.of(
-            AMOUNT_KEY, (amount) -> {
-                Integer amountValue = Integer.valueOf(amount);
-                return MIN_AMOUNT <= amountValue && MAX_AMOUNT >= amountValue;
-            }
-    );
+    private final static Map<String, Function<String, Boolean>> validators =
+            ImmutableMap.<String, Function<String, Boolean>>builder()
+                    .put(AMOUNT_KEY, (amount) -> {
+                        Integer amountValue = Integer.valueOf(amount);
+                        return MIN_AMOUNT <= amountValue && MAX_AMOUNT >= amountValue;
```

Comment:
I guess that this technically could be a size validator (with max / min size, as you did before) but I guess we can keep it like this for now